<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OverviewPage</class>
 <widget class="QWidget" name="OverviewPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>756</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_blue">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>981</width>
        <height>220</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>960</width>
        <height>220</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>0</width>
        <height>500</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QFrame" name="frame_blue_1">
         <property name="minimumSize">
          <size>
           <width>345</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="frame_blue_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>345</width>
            <height>201</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>345</width>
            <height>200</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <property name="spacing">
                <number>6</number>
               </property>
               <property name="sizeConstraint">
                <enum>QLayout::SetFixedSize</enum>
               </property>
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>Balance</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="labelWalletStatus">
                 <property name="minimumSize">
                  <size>
                   <width>0</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>24</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>(out of sync)</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QLabel" name="labelTotalText">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>15</height>
                </size>
               </property>
               <property name="text">
                <string>Total:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="labelTotal">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>100</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>100</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>25</pointsize>
                 <weight>50</weight>
                 <bold>false</bold>
                </font>
               </property>
               <property name="mouseTracking">
                <bool>false</bool>
               </property>
               <property name="text">
                <string>0.00</string>
               </property>
               <property name="scaledContents">
                <bool>true</bool>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_11">
               <item>
                <spacer name="horizontalSpacer_8">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="blabel_esbcoin">
                 <property name="font">
                  <font>
                   <pointsize>20</pointsize>
                   <italic>false</italic>
                   <kerning>true</kerning>
                  </font>
                 </property>
                 <property name="cursor">
                  <cursorShape>PointingHandCursor</cursorShape>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="flat" stdset="0">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_blue_2">
         <property name="minimumSize">
          <size>
           <width>220</width>
           <height>200</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QLabel" name="labelBalanceText">
              <property name="text">
               <string>Available:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="labelBalance">
                <property name="text">
                 <string>0.00</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="labelPendingText">
              <property name="text">
               <string>Pending:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="labelUnconfirmed">
                <property name="text">
                 <string>0.00</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="labelImmatureText">
              <property name="text">
               <string>Immature:</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="labelWatchImmature">
                <property name="text">
                 <string>0.00</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="minimumSize">
          <size>
           <width>330</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>150</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">border-image: url(':/images/light/helpthehomeless_logo_horizontal') 0 0 0 0 stretch stretch;
        border-style: none;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QFrame" name="frameBlockchainInfo_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>220</y>
        <width>451</width>
        <height>311</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: transparent</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_19">
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>41</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14">
         <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">border: transparent;
font-size: 20px;</string>
           </property>
           <property name="text">
            <string>Block Height</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>29</height>
            </size>
           </property>
          </spacer>
         </item>
         <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
          <widget class="QLabel" name="label_24hblockchaininfo_1">
           <property name="styleSheet">
            <string notr="true">border: transparent;
font-size: 20px;</string>
           </property>
           <property name="text">
            <string>Network Hashrate</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="frameBlockchainInfoData_3">
         <property name="styleSheet">
          <string notr="true">border: transparent;
; font-size: 15px;</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <item row="3" column="2" alignment="Qt::AlignRight|Qt::AlignVCenter">
           <widget class="QLabel" name="label_Nethash_3">
            <property name="styleSheet">
             <string notr="true">font-size: 15px;</string>
            </property>
            <property name="text">
             <string>Nethash MHs:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_CurrentBlock">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 15px</string>
            </property>
            <property name="text">
             <string>Current block:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="3" alignment="Qt::AlignRight">
           <widget class="QLabel" name="label_Nethash_value_3">
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_CurrentBlock_value_3">
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>130</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 15px;</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>135</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_4">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>430</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit HTH Partners and Donate Today!!!!!!!!</string>
      </property>
      <property name="styleSheet">
       <string notr="true">&lt;a href=&quot;https://hth.world/partners.html&quot;&gt;&lt;/a&gt;</string>
      </property>
      <property name="text">
       <string>Partners</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_1">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>350</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="sizeIncrement">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit Help The Homeless Coin Website</string>
      </property>
      <property name="styleSheet">
       <string notr="true">&lt;a href=&quot;https://hth.world&quot;&gt;&lt;/a&gt;</string>
      </property>
      <property name="text">
       <string>HTH Coin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>350</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit Help The Homeless Worldwide A NJ Nonprofit Corporation</string>
      </property>
      <property name="styleSheet">
       <string notr="true">&lt;a href=&quot;https://www.helpthehomelessworldwide.org&quot;&gt;&lt;/a&gt;</string>
      </property>
      <property name="text">
       <string>Nonprofit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_3">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>430</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>Visit Help The Homeless Coin Explorer and Track Your HTH Coins</string>
      </property>
      <property name="styleSheet">
       <string notr="true">&lt;a href=&quot;https://chainz.cryptoid.info/hth&quot;&gt;&lt;/a&gt;</string>
      </property>
      <property name="text">
       <string>Explorer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Website_2">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>350</y>
        <width>120</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>30</height>
       </size>
      </property>
      <property name="toolTip">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Visit AltMarkets Exchange and Trade Help The Homeless Coin Today!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true">&lt;a href=&quot;https://hth.world/exchanges.html&quot;&gt;&lt;/a&gt;</string>
      </property>
      <property name="text">
       <string>AltMarkets</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>280</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;</string>
      </property>
      <property name="text">
       <string>HTH Worldwide</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>280</y>
        <width>91</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;</string>
      </property>
      <property name="text">
       <string>HTH Coin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>280</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 20px;</string>
      </property>
      <property name="text">
       <string>HTH Exchanges</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
